internal/generator/vector/output/loki/loki.go

```diff
@@ -220,9 +215,26 @@ func TLSConf(o logging.OutputSpec, secret *corev1.Secret, op Options) []Element
 			tlsConf,
 		}
 	}
+	if o.Secret != nil || (o.TLS != nil && o.TLS.InsecureSkipVerify) {
+
+		if tlsConf := security.GenerateTLSConf(o, secret, op, false); tlsConf != nil {
+			tlsConf.NeedsEnabled = false
+			return []Element{tlsConf}
+		}
+	}
+
 	return []Element{}
 }
 
+func isDefaultOutput(inputs []string, name string) bool {
+	for _, input := range inputs {
+		if lokistack.FormatOutputNameFromInput(input) == name {
+			return true
+		}
+	}
+	return false
```

Comment:
This does not seem to work as expected (it seems like my last test did not include the latest refactor yet, I'm sorry for that). The "input names" in `inputs` are already preprocessed, and don't match the `input.name` values used to originally create the output name, so the comparison will never be true (not even for custom names).

I'm not sure of the unintended consequences, but in the test-cases I created just checking for a `default-` prefix in the output name seems to work:

```suggestion
	return strings.HasPrefix(name, "default-")
```

Here's [a branch](https://github.com/jcantrill/cluster-logging-operator/compare/log4476...xperimental:cluster-logging-operator:log4476) containing the change.